api/object.go

```diff
@@ -261,7 +261,7 @@ func GetObjectReader(bp BaseParams, bck cmn.Bck, objName string, args *GetArgs)
 func (args *PutArgs) getBody() (io.ReadCloser, error) { return args.Reader.Open() }
 
 func (args *PutArgs) put(reqArgs *cmn.HreqArgs) (*http.Request, error) {
-	req, err := reqArgs.ReqDeprecated() // TODO: deprecated; use reqArgs.Req()
+	req, err := reqArgs.Req()
```

Comment:
any chance you could stress test this path? Use aisloader with `putpct=100`, compare with and without, let us know..

PS. the only reason I did not remove these bits is exactly that - testing